deployment/copilot/templates/traffic-replayer/manifest.yml

```diff
@@ -33,6 +33,10 @@ memory: 4096    # Amount of memory in MiB used by the task.
 count: 1       # Number of tasks that should be running in your service.
 exec: true     # Enable getting a shell to your container (https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-exec.html).
 
+# Pass environment variables as key value pairs.
+variables:
+  TUPLE_DIR_NAME: SERVICE_NAME_PLACEHOLDER
```

Comment:
See the comment up above - this could just be set to include the root name too (& we can default the env variable in log4j to just ./logs/ or something like that)